README.md

```diff
@@ -20,7 +20,10 @@ Ensure you have docker installed and increase the memory allocation to at least
 
     `127.0.0.1       activiti-cloud-sso-idm`
 2. `cd docker/`
-    1. Run > `docker-compose -f infrastructure-docker.yml up -d` (to start all the infrastructure services) 
+    1. Run > `docker-compose -f infrastructure-docker.yml up -d` (to start all the infrastructure services except audit service) 
```

Comment:
@daisuke-yoshimoto I was discussing with @Salaboy and @ryandawsonuk about the best way to provide the mongo db implementation inside the docker compose file. We would like to avoid to add an extra step in order to start all the services. What we propose is the following:
1. keep the current section for `activiti-cloud-audit`using JPA implementation in the file `infrastructure-docker.yml`.
2. Add the mongo section to the same file, but commented out.
3. If someone wants to use the Mongo implementation instead of the JPA one, he needs to uncomment the Mongo section and comment out the JPA one.

What do you think?